src/Sulu/Bundle/SecurityBundle/Controller/ResettingController.php

```diff
@@ -224,11 +236,25 @@ public function sendEmailAction(Request $request)
         try {
             /** @var UserInterface $user */
             $user = $this->findUser($request->get('user'));
+        } catch (EntityNotFoundException $ex) {
+            // do not report when there is no user, because this could be used for username enumeration attacks
+            return new JsonResponse(null, Response::HTTP_NO_CONTENT);
+        } catch (UserNotInSystemException $ex) {
+            if ($this->logger instanceof LoggerInterface) {
+                $this->logger->error($ex->getMessage(), ['exception' => $ex]);
+            }
+            $response = new JsonResponse(null, Response::HTTP_BAD_REQUEST);
+        }
+
+        try {
```

Comment:
This is expected in all cases the same http status code and content need to be returned: 

```suggestion
```

https://github.com/sulu/sulu/security/advisories/GHSA-wfm4-pq59-wg6r